core/templates/dev/head/filters.js

```diff
@@ -165,12 +165,11 @@ oppia.filter('parameterizeRuleDescription', [
       console.error('Cannot find interaction with id ' + interactionId);
       return '';
     }
-
     var description = INTERACTION_SPECS[interactionId].rule_descriptions[
-      rule.rule_type];
+      rule.type];
     if (!description) {
       console.error(
-        'Cannot find description for rule ' + rule.rule_type +
+        'Cannot find description for rule ' + rule.ruleType +
```

Comment:
rule.type?

(Please grep for residual instances of ruleType, and other strings you've changed)